src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  cursor: text;
+}
+
+.search {
+  outline: 0;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.style {
+  width: 100%;
+  height: 100%;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0,0,0,0.25);
+  font-weight: 800;
+}
+
+.search:focus::placeholder {
+  text-shadow: 0 4px 4px rgba(0,0,0,0.25);
+  font-weight: 800;
+}
+
+.form--small {
+  position: relative;
+  margin-top: 20px;
+  height: 42px;
+}
+
+.img--small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  cursor: text;
+}
+
+.search--small {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
```

Comment:
Move it to `html` selector and remove it from all other places.

Reply:
font-family: Avenir, sans-serif;
Moved the selector to html and the font stopped working

Reply:
> Move it to `html` selector and remove it from all other places.

font-family: Avenir, sans-serif;
Moved the selector to html and the font stopped working